speeduino/scheduler.h

```diff
@@ -173,8 +177,9 @@ struct Schedule {
 
   COMPARE_TYPE nextStartCompare;      ///< Planned start of next schedule (when current schedule is RUNNING)
   COMPARE_TYPE nextEndCompare;        ///< Planned end of next schedule (when current schedule is RUNNING)
-  volatile bool hasNextSchedule = false; ///< Enable flag for planned next schedule (when current schedule is RUNNING)
-  volatile bool endScheduleSetByDecoder = false;
+
+  volatile byte scheduleFlags;        ///< All flags used in the scheduler
```

Comment:
```suggestion
 volatile  uint8_t hasNextSchedule :1;
 volatile  uint8_t endScheduleSetByDecoder :1;
 volatile  uint8_t scheduleRepeatIsActive :1;
 volatile  uint8_t ignitionRepeatCount :5;
```
This should save even more ram as repeat count can also be squeezed into the same byte with flags. Also flags retain their original usage style in the code, so sadly, yes then you would have to revert all that too.
(Note 'ignRprCnt' renamed to 'ignitionRepeatCount' this is out of pure respect for others reading the code, not having to figure what 'ignceint reptile canteen' or what it is.)